app/lib/efile/az/az140_calculator.rb

```diff
@@ -255,7 +263,7 @@ def calculate_line_53
         # AZ income tax withheld
         # sum of tax withheld from all income documents: W-2, 1099-R, 1099-G, 1099-INT
         @direct_file_data.total_w2_state_tax_withheld +
-          @intake.state_file1099_gs.sum(&:state_income_tax_withheld) +
+          @intake.state_file1099_gs.sum(&:state_income_tax_withheld_amount).round +
```

Comment:
do we want to 1. round and sum or 2. sum then round?

Reply:
i think we should round and then sum!